src/main/java/ch/njol/skript/effects/EffReturn.java

```diff
@@ -131,6 +126,11 @@ protected void execute(Event event) {
 		assert false;
 	}
 
+	@Override
+	public ExecutionIntent executionIntent() {
+		return ExecutionIntent.stopTrigger();
```

Comment:
Do you think this will always be true with the return api? What about code like:
```
blah blah structure:
  trigger:
    ...
    my special section:
      ...
      return {_value}
    # code continues
```
This is more like stops 1 section, right?

Reply:
The return effect always stops the trigger. This might be an oversight with the return api :(

Reply:
Oh yeah I see what you're saying. This is fine for now, but we might want to re-examine what happens after a return